### PR TITLE
[wpmlcore-5173] Created WPML_VC_Website_Builder_Update_Translation

### PR DESCRIPTION
**Pull request by adrianosferreira**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
@adrianosferreira looks good +1

File: src/class-wpml-vc-website-builder-update-translation.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		}
+	}
+
+	/**
+	 * @param string $node_id
+	 * @param $settings
+	 *
+	 * @return mixed
+	 */
+	public function update_strings_in_node( $node_id, $settings ) {
```
Comment (brucepearson) [inline src/class-wpml-vc-website-builder-update-translation.php]: @adrianosferreira This code is a duplicate of the code in BB.

Comment (adrianosferreira) [reply src/class-wpml-vc-website-builder-update-translation.php]: @brucepearson you are right. I will move it to the existing parent class.

**Comment by adrianosferreira** [comment]
@BruceOnTheGo please review it.

File: tests/phpunit/util/wpml-pb-test-case2.php [hunk 2]
```diff
@@ -0,0 +1,191 @@
+<?php
+
+abstract class WPML_PB_TestCase2 extends OTGS_TestCase {
```
Comment (brucepearson) [inline tests/phpunit/util/wpml-pb-test-case2.php]: @adrianosferreira This is a copy of the file from pagebuilders. I think there must be something wrong with the way we are structuring the code if we need to do this.

Comment (adrianosferreira) [reply tests/phpunit/util/wpml-pb-test-case2.php]: @brucepearson actually I don't need this, removed.

**Review by brucepearson — APPROVED** [review]
@adrianosferreira This looks ok from what I can see.

**Comment by adrianosferreira** [comment]
Merged manually, closing.

**Pull request closed without merging**